src/test/resources/join_objects.json

```diff
@@ -6,13 +6,13 @@
 {"messageTile": "c", "join_field": "parentType"}
 {"index":{"_type": "joinType", "_id":"4"}}
 {"messageTile": "d", "join_field": "parentType"}
-{"index":{"_type": "joinType", "_id":"5", "_routing": "1"}}
+{"index":{"_type": "joinType", "_id":"5", "routing": "1"}}
```

Comment:
Why renaming _routing to routing?

Reply:
This is the breaking changes in ES 7.0. [The _routing with underscore has been removed.](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#_getdate_and_getdates_removed)